ibverbs/src/lib.rs

```diff
@@ -377,17 +377,23 @@ impl Context {
             }
         }
 
-        // let mut gid = ffi::ibv_gid::default();
-        let mut gid = Gid::default();
-        let ok = unsafe { ffi::ibv_query_gid(ctx, PORT_NUM, 0, gid.as_mut()) };
-        if ok != 0 {
-            return Err(io::Error::last_os_error());
-        }
+        let mut gid_table = vec![ffi::ibv_gid_entry::default(); port_attr.gid_tbl_len as usize];
+        let num_entries = unsafe {
+            ffi::_ibv_query_gid_table(
```

Comment:
How come this function starts with a leading `_`? Is it an internal or unstable function somehow?